spec/components/admin/participants/table_row_spec.rb

```diff
@@ -12,21 +12,9 @@
 
   context "when profile is associated with the school" do
     let(:school) { create :school }
-    let(:participant_profile) { create :participant_profile, school: school }
+    let(:participant_profile) { create :ect_participant_profile, school: school }
 
     it { is_expected.to have_content school.name }
     it { is_expected.to have_content school.urn }
   end
-
-  context "when profile is not associated with the school" do
-    let(:participant_profile) { create :participant_profile, school: nil }
-
-    it { is_expected.to have_css(".govuk-table__cell:nth-child(3)", text: "\n    School\n    \n") }
-  end
-
-  context "when profile is not associated with the school but has school_urn" do
-    let(:participant_profile) { create :participant_profile, school: nil, school_urn: "123456" }
-
-    it { is_expected.to have_content("123456") }
-  end
```

Comment:
Notes: I think this tests weren't very useful as this case can not really happen. can't have a `ParticipantProfile(::ECF)` without a school